src/app/features/user/active-user/active-user.component.html

```diff
@@ -0,0 +1,15 @@
+<mat-card>
+    <mat-card-content>
+        <div *ngIf="loading" style="text-align: center;">
+            <h3>Please waiting for activating your account...</h3>
+        </div>
+        <div *ngIf="isActived && !loading" style="text-align: center;">
+            <h3>Account Successfully Activated!</h3>
+            <button mat-raised-button routerLink='/auth/login'>Sign In Now</button>
+        </div>
+        <div *ngIf="!isActived && !loading" style="text-align: center;">
+            <h3>The active link is invalid! Please check again.</h3>
```

Comment:
This activation link is invalid. Please try again.

Reply:
updated, @1ceb3rg thanks